ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/private/provision/cucushift-installer-rehearse-aws-ipi-private-provision-chain.yaml

```diff
@@ -4,10 +4,17 @@ chain:
     - ref: ipi-conf
     - ref: aws-provision-vpc-shared
```

Comment:
Why not just "aws-provision-vpc"?

Reply:
following some kinds of manner, we call it _**customer/existing/shared**_ VPCs